Client/core/Graphics/CMaterialPrimitive3DBatcher.cpp

```diff
@@ -0,0 +1,243 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        CMaterialPrimitive3DBatcher.cpp
+ *  PURPOSE:
+ *
+ *
+ *****************************************************************************/
+#include <StdInc.h>
+#include "CMaterialPrimitive3DBatcher.h"
+////////////////////////////////////////////////////////////////
+//
+// CMaterialPrimitive3DBatcher::CMaterialPrimitive3DBatcher
+//
+//
+//
+////////////////////////////////////////////////////////////////
+CMaterialPrimitive3DBatcher::CMaterialPrimitive3DBatcher(bool bPreGUI, CGraphics* graphics)
+{
+    m_bPreGUI = bPreGUI;
+    m_pGraphics = graphics;
+}
+////////////////////////////////////////////////////////////////
+//
+// CMaterialPrimitive3DBatcher::~CMaterialPrimitive3DBatcher
+//
+//
+//
+////////////////////////////////////////////////////////////////
+CMaterialPrimitive3DBatcher::~CMaterialPrimitive3DBatcher(void)
```

Comment:
Remove the destructor from header and translation unit if you don't do any work here.